domains/emissions/Emissions.API/API/Program.cs

```diff
@@ -19,37 +22,53 @@
 builder.Logging.ClearProviders();
 builder.Logging.AddSerilog(logger);
 
-// Add services to the container.
+builder.Services.AddHealthChecks().AddAsyncCheck("Configuration check", () =>
+{
+    try
+    {
+        Configuration.GetDataSyncEndpoint();
+        Configuration.GetEnergiDataServiceEndpoint();
+        Configuration.GetRenewableSources();
+        Configuration.GetWasteRenewableShare();
+        return Task.FromResult(HealthCheckResult.Healthy());
+    }
+    catch
+    {
+        return Task.FromResult(HealthCheckResult.Unhealthy());
+    }
+});
+
 builder.Services.AddControllers().AddJsonOptions(options =>
 {
+    options.JsonSerializerOptions.Converters.Add(new CustomJsonStringEnumConverter<QuantityUnit>());
     options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
     options.JsonSerializerOptions.PropertyNamingPolicy = JsonNamingPolicy.CamelCase;
 });
 
-builder.Services.AddValidatorsFromAssemblyContaining<EnergySourceRequest.Validator>();
+builder.Services.AddValidatorsFromAssemblyContaining<Program>(lifetime: ServiceLifetime.Scoped);
+builder.Services.AddFluentValidationAutoValidation();
 
-// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
 builder.Services.AddSwaggerGen();
 
-// Inform Swagger about FluentValidation rules. See https://github.com/micro-elements/MicroElements.Swashbuckle.FluentValidation for more details
-builder.Services.AddTransient<IValidatorFactory, ServiceProviderValidatorFactory>();
 builder.Services.AddFluentValidationRulesToSwagger();
 
-builder.Services.AddHttpClient();
-builder.Services.AddCustomServices();
+builder.Services.AddHttpClient<IEnergiDataService, EnergiDataService>(x => x.BaseAddress = new Uri(Configuration.GetEnergiDataServiceEndpoint()));
```

Comment:
Register in extension method. 1 for data sync and one for emission service.

Reply:
We discussed these one line configurations/setups to an/separate extension method(s) and opted to keep it in `Program.cs`.